.ci/docker/manywheel/build_scripts/build.sh

```diff
@@ -117,6 +117,8 @@ find /opt/_internal \
   -print0 | xargs -0 rm -f
 
 for PYTHON in /opt/python/*/bin/python; do
+    # install setuptools since python 3.12 its required to use distutils
+    $PYTHON -m pip install setuptools
```

Comment:
Please do not install unpinned dependencies.... Specify the version you want to install here